app/airq/commands/__init__.py

```diff
@@ -20,6 +22,8 @@
     ShowAboutHandler.route(pattern=r"^3[\.\)]?$"),
     ShowMenuHandler.route(pattern=r"^m$"),
     StopHandler.route(pattern=r"^u$"),
+    ShowFeedbackHandler.route(pattern=r"^f$"),
+    RecieveFeedbackHandler.route(pattern=r"a^"),
```

Comment:
Can we make this a bit clearer by making this a constant? E.g.,

```python3
MATCH_NONE = r"a^"

# later...

RecieveFeedbackHandler.route(pattern=MATCH_NONE)
```

We should think of ways to make this pattern suck less. 🤷‍♂️

Reply:
Also, it's Receive, not Recieve.